### PR TITLE
[TASK] better error when bridge down (RT-2147)

### PR DESCRIPTION
**Pull request by darkdarkdragon**
Send: when bridge are down, show error "could not contact bridge"
instead of "This address is not Ripple-enabled."

### CODE REVIEW

**Comment by darkdarkdragon** [comment]
rt-2147

**Comment by jublo** [comment]
- Can you insert a line break before `Please`?
- I'd like to see some spacing between the lines `try again later` and `Recipient will receive`.

File: src/js/services/federation.js [hunk 1]
```diff
@@ -37,56 +49,58 @@ module.factory('rpFederation', ['$q', '$rootScope', 'rpRippleTxt',
...
+                url: txt.federation_url[0]
               });
             });
           },
           success: function (data) {
             $scope.$apply(function() {
-              if ("object" === typeof data &&
-                  "object" === typeof data.federation_json &&
-                  data.federation_json.type === "federation_record" &&
+              if ('object' === typeof data &&
```
Comment (jublo) [inline src/js/services/federation.js]: Should the `&&` signs not be at the beginning of the lines?

File: src/js/tabs/send.js [hunk 2]
```diff
@@ -219,18 +223,25 @@ SendTab.prototype.angular = function (module)
               $scope.sendForm.send_destination.$setValidity("federation", false);
```
Comment (jublo) [inline src/js/tabs/send.js]: While you're at cleaning up the code style, why not use single quotes here, too? :wink:

Comment (darkdarkdragon) [inline src/js/tabs/send.js]: hm. missed it somehow :)

File: src/js/services/federation.js (hunk 1 above)
Comment (darkdarkdragon) [inline src/js/services/federation.js]: Can't find this requirement in any style guide - can you point me out?

**Comment by darkdarkdragon** [comment]
rt-2147-2

**Comment by rippletravis** [comment]
Visuals look good.

**Comment by mrajvanshy** [comment]
LGTM

**Comment by MatthewPhinney** [comment]
LGTM

**Comment by vhpoet** [comment]
LGTM

**Pull request merged**